Userland/Libraries/LibWeb/FileAPI/File.cpp

```diff
@@ -106,7 +106,7 @@ WebIDL::ExceptionOr<void> File::serialization_steps(HTML::SerializationRecord& r
     TRY(HTML::serialize_string(vm, record, m_name));
```

Comment:
ah something i forgot to comment on your previous PR

line 94 here you serialize the interface name in the serialization steps. but in the deserialization steps, the structured_deserialize logic has stripped the interface name from the serialization record before it hands it off to the object type. So, we should probably make this consistent. I would recommend having the serialization logic serialize `serializable->interface_name()` before calling its serialization steps.